packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -52,7 +52,7 @@ abstract class CameraPlatform extends PlatformInterface {
   }
 
   /// Initializes the camera on the device.
-  Future<void> initializeCamera(int cameraId) {
+  Future<void> initializeCamera(int cameraId, {int imageStreamImageFormat}) {
```

Comment:
nit: simplify argumentname to `imageFormatGroup`

It would also be nice to update the API documentation here to mention the optional `imageFormatGroup` and list the default values in case you didn't specify any.